### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Kraviecc**
#39

### CODE REVIEW

File: CocoJumper/Commands/CocoJumperCommandPackage.cs [hunk 1]
```diff
@@ -25,6 +25,7 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
             await CocoJumperSingleSearchCommand.InitializeAsync(this);
             await CocoJumperSingleSearchHighlightCommand.InitializeAsync(this);
             await base.InitializeAsync(cancellationToken, progress);
+            await CocoJumper.Commands.CocoJumperWordSearchCommand.InitializeAsync(this);
```
Comment (C485) [inline CocoJumper/Commands/CocoJumperCommandPackage.cs]: Move this line one up

File: CocoJumper/Commands/CocoJumperWordSearchCommand.cs [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+            CocoJumperCommandPackage cocoJumperCommandPackage = (CocoJumperCommandPackage)_package;
+
+            CleanupLogicAndInputListener();
+            WpfViewProvider renderer = new WpfViewProvider(wpfTextView);
+
+            _logic = new CocoJumperLogic(renderer, cocoJumperCommandPackage);
+            _inputListener = new InputListener(textView);
+            _inputListener.KeyPressEvent += OnKeyboardAction;
+            _logic.ActivateSearching(false, false, true);
+            OnKeyboardAction(this, null, KeyEventType.KeyPress);
```
Comment (C485) [inline CocoJumper/Commands/CocoJumperWordSearchCommand.cs:42]: It's necessary?

**Review by C485 — CHANGES_REQUESTED** [review]
Move this one line up and it's ready

File: CocoJumper/Commands/CocoJumperWordSearchCommand.cs (hunk 2 above)
Comment (Kraviecc) [reply CocoJumper/Commands/CocoJumperWordSearchCommand.cs:42]: Yes, it's auto-searching mode where you don't literally press additional key to invoke searching. So we have to immediately invoke searching through view when pressing CTRL + ALT + ' (or rather parsing view to retrieve words and letters connected with them) and render markers.

**Review by C485 — DISMISSED** [review]

**Review by C485 — APPROVED** [review]

**Comment by Kraviecc** [comment]
Closes #39

**Pull request merged**